tgui/packages/tgui/interfaces/PlantAnalyzer.tsx

```diff
@@ -0,0 +1,577 @@
+
+            <LabeledList.Item label="Weeds">
+              {seed_data.weed_chance && seed_data.weed_rate
+                ? seed_data.weed_chance +
+                  '% chance to grow by ' +
+                  seed_data.weed_rate
+                : 'No weed growth'}
+            </LabeledList.Item>
+          </LabeledList>
+        </Stack.Item>
+      </Stack>
+    </Section>
+  );
+};
+
+export const SeedExtraData = (props) => {
+  const { data } = useBackend<PlantAnalyzerData>();
+  const { seed_data } = data;
+  return (
+    <>
+      {!!seed_data.graft_gene && (
+        <TraitTooltip
+          path={seed_data.graft_gene}
+          trait_db={data.trait_db}
+          grafting
+        />
+      )}
+
+      {seed_data.removable_traits?.map((trait) => (
+        <TraitTooltip key="" path={trait} trait_db={data.trait_db} removable />
+      ))}
+      {seed_data.core_traits?.map((trait) => (
+        <TraitTooltip key="" path={trait} trait_db={data.trait_db} />
+      ))}
+      {!!seed_data.mutatelist.length && (
+        <Tooltip content={`Mutates into: ${seed_data.mutatelist.join(', ')}`}>
+          <Icon name="dna" m={0.5} />
+        </Tooltip>
+      )}
+      {!!seed_data.juice_name && (
+        <Tooltip content={`Juicing result: ${seed_data.juice_name}`}>
+          <Icon name="glass-water" m={0.5} />
+        </Tooltip>
+      )}
+      {!!seed_data.distill_reagent && (
+        <Tooltip content={`Ferments into: ${seed_data.distill_reagent}`}>
+          <Icon name="wine-bottle" m={0.5} />
+        </Tooltip>
+      )}
+      {seed_data.reagents.length > 0 && (
+        <Tooltip
+          content={
+            <ReagentTooltip
+              reagents={seed_data.reagents}
+              grind_results={seed_data.grind_results}
+              potency={seed_data.potency}
+              volume_mod={seed_data.volume_mod}
+            />
+          }
+        >
+          <Icon name="blender" m={0.5} />
+        </Tooltip>
+      )}
+    </>
+  );
+};
+
+export const PlantAnalyzerGraft = (props) => {
+  const { act, data } = useBackend<PlantAnalyzerData>();
+  const { graft_data } = data;
+  return (
+    <Section
+      title={'Graft: ' + capitalizeFirst(graft_data.name)}
+      buttons={
+        !!graft_data.graft_gene && (
+          <TraitTooltip path={graft_data.graft_gene} trait_db={data.trait_db} />
+        )
+      }
+    >
+      <Stack>
+        <Stack.Item ml={2} mr={4}>
+          <DmIcon
+            fallback={fallback_big}
+            icon={graft_data.icon}
+            icon_state={graft_data.icon_state}
+            height="64px"
+            width="64px"
+          />
+        </Stack.Item>
+        <Stack.Item>
+          <LabeledList>
+            <LabeledList.Item label="Endurance">
+              <ProgressBar
+                value={graft_data.endurance / 100}
+                ranges={{
+                  good: [0.7, Infinity],
+                  average: [0.3, 0.7],
+                  bad: [0, 0.3],
```

Comment:
Most plants sit towards the lower end so we should adjust the ranges to accommodate this 
(50 endurance is quite good, and only like three plants have stats above that by default - meanwhile 25-30 endurance is what I would consider "pretty average", as 15 is the default) 
```suggestion
                  good: [0.5, Infinity],
                  average: [0.1, 0.5],
                  bad: [0, 0.1],
```